pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ArithmeticFunctions.java

```diff
@@ -104,4 +111,30 @@ public static double ln(double a) {
   public static double sqrt(double a) {
     return Math.sqrt(a);
   }
+
+  @ScalarFunction
+  public static double sign(double a) {
+    return Math.signum(a);
+  }
+
+  @ScalarFunction
+  public static double pi() { return Math.PI; }
+
+  @ScalarFunction
+  public static double e() { return Math.E; }
```

Comment:
Constants should be implemented with `LiteralTransformFunction` so they can be shared between segments.

Reply:
The reason I had to do it this way because we want to integrate it with Tableau and Tableau expects these as functions e.g. PI() instead of PI.

Reply:
Sure, it needs to be a function, but it should be implemented as or similar to LiteralTransformFunction so that the array of e or pi values can be stored in the query context and shared across segments. Having said that, these will be very rarely invoked functions so maybe this can be deferred until someone actually complains about them.

Reply:
These functions without argument should be recognized as literal only expression in broker request handler, and replaced with the actual literal value, so in the query context it should already be overridden to literal